server/src/DBTablesMonitoring.cc

```diff
@@ -2540,6 +2540,23 @@ HatoholError DBTablesMonitoring::getNumberOfMonitoredItemsPerSecond
 	return HatoholError(HTERR_OK);
 }
 
+size_t DBTablesMonitoring::getNumberOfEvents(const EventsQueryOption &option)
+{
+	DBClientJoinBuilder builder(tableProfileEvents, &option);
+	string stmt =
```

Comment:
Hmm..., if we make this value to `constexpr`, its type should be `char` instead of `std::string`.